cypress/test/search-field.spec.ts

```diff
@@ -0,0 +1,69 @@
+import SearchFieldSteps from "../../cypress/steps/search-field-steps";
+
+describe('Onto Search Results Component', () => {
+  it('Should find the component', () => {
+    SearchFieldSteps.visit();
+    SearchFieldSteps.getSearchFieldComponent().should('be.visible');
+  });
+
+  it('Should autocomplete', () => {
+    SearchFieldSteps.visit();
+    SearchFieldSteps.typeSearchPhrase('D');
+    SearchFieldSteps.getAutocompleteLabel(0).should('be.visible').should('contain', 'Dehydroepiandrosterone');
+    SearchFieldSteps.getAutocompleteType(0).should('be.visible').should('contain', 'small molecule');
+    SearchFieldSteps.getAutocompleteLabels(0).should('be.visible').should('contain', 'Prasterone,5-Dehydroepiandrosterone,DHEA,5-DHEA,3-beta-Hydroxy-5-androsten-17-one,DHA');
+  });
+
+  it('Should make visual difference when free text or autocomplete is selected', () => {
+    SearchFieldSteps.visit();
+    SearchFieldSteps.typeSearchPhrase('D');
+    SearchFieldSteps.getAutocompleteLabel(0).click();
+    SearchFieldSteps.completeSearchPhrase('free text');
+
+    SearchFieldSteps.getStateSelection(0).should('be.visible')
+      .should('contain', 'Dehydroepiandrosterone')
+      .should('have.css', 'background-color', 'rgb(173, 216, 230)');
+
+    SearchFieldSteps.getStateSelection(1).should('be.visible')
+      .should('contain', 'free text')
+      .should('have.css', 'background-color', 'rgb(224, 224, 224)');
```

Comment:
Can't we have a css class for finding the free text selection and the regular once?

Reply:
I just saw that you you actually have such classes. Why don't you just check for the class and not for these colors here? It's more fragile as it is now.